contributors/contributor-entitlement-program.md

```diff
@@ -0,0 +1,15 @@
+# Contributor Entitlement Program
+
+Continuing our commitment to work in partnership with the Community, Chef offers a Contributor Entitlement program.  At Chef, we recognize and value the Community’s involvement and we are dedicated to enabling our contributors to use the very software they are helping to build.
```

Comment:
Is community capitalized @chef/docs-team ?

Reply:
@tas50 No, not in this case because it is a generic label. If you did "Chef Community," it would be capitalized as it is a proper noun and formal name. Does that make sense?

Reply:
It does - fixing now!